lib/overcommit/hook_context/pre_push.rb

```diff
@@ -18,9 +18,21 @@ def pushed_refs
     end
 
     def modified_files
-      @modified_files ||= Overcommit::GitRepo.modified_files(
-        refs: "#{pushed_refs[0].remote_sha1}..#{pushed_refs[0].local_sha1}"
-      )
+      @modified_files ||= Overcommit::GitRepo.modified_files(refs: refs)
+    end
+
+    def modified_lines_in_file(file)
+      @modified_lines ||= {}
+      unless @modified_lines[file]
+        @modified_lines[file] =
+          Overcommit::GitRepo.extract_modified_lines(file, refs: refs)
+      end
+
+      @modified_lines[file]
+    end
+
+    def refs
```

Comment:
Let's use the nomenclature in the [Git documentation](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#_commit_ranges) and call this `ref_range`. I'm concerned that in this context `refs` is a bit too vague for folks looking at this code for the first time.

Reply:
Agreed.